packages/tooltip/src/index.test.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { render, cleanup, fireEvent, act } from "@testing-library/react";
+import Tooltip, { LEAVE_TIMEOUT, MOUSE_REST_TIMEOUT } from ".";
+
+jest.mock("@reach/utils", () => ({
+  ...jest.requireActual("@reach/utils"),
+  checkStyles: jest.fn()
+}));
+
+jest.useFakeTimers();
+
+describe("Tooltip", () => {
+  beforeEach(() => {
+    cleanup();
```

Comment:
You don't need this in the latest version of React Testing Library, it does it automatically.

Reply:
you are right 👍

Reply:
Note to anyone not following the conversation: you do need it in the current setup with the re-require of the modules in beforeEach